api/novelist.py

```diff
@@ -391,23 +394,19 @@ def process_item(self, identifier):
             self.output_source, metadata.primary_identifier,
             strength=1
         )
-        # Create an edition with the NoveList metadata and identifier
+        # Create an edition with the NoveList metadata & NoveList identifier.
+        # This will capture equivalent ISBNs and less appealing metadata in
+        # its unfettered state on the NoveList identifier alone.
         edition, ignore = metadata.edition(self._db)
         metadata.apply(edition)
+
         if edition.series or edition.series_position:
-            # Place any missing series data that NoveList gathered into this
-            # identifier's edition.
-            self._apply_series_data(
-                identifier, edition.series, edition.series_position
-            )
+            metadata.primary_identifier = identifier
```

Comment:
Extend your comment to make it explicit that you're modifying the Metadata you got from Novelist so that it can be applied directly to the original identifier.